core/widgetevents.go

```diff
@@ -467,14 +467,14 @@ func (wb *WidgetBase) HandleClickOnEnterSpace() {
 	})
 }
 
-///////////////////////////////////////////////////////////////////
-//		Focus
+////////	Focus
 
 // SetFocus sets the keyboard input focus on this item or the first item within it
 // that can be focused (if none, then just sets focus to this object).
 // This does not send an [events.Focus] event, which typically results in
 // the widget being styled as focused. See [WidgetBase.SetFocusEvent] for
-// a version that does.
+// a version that does. Also see [WidgetBase.StartFocus] which must be
+// called instead during initial construction prior to the initial scene render.
 func (wb *WidgetBase) SetFocus() {
```

Comment:
Would any end user call `SetFocus` instead of `SetFocusEvent`? Should this be unexported and `SetFocusEvent` get a better name?

Reply:
Going with `SetFocusQuiet` instead of `SetFocus` and `SetFocus` instead of `SetFocusEvent`, since quiet is rarer and thus should be the one with the suffix.